src/component/books/Showbooks.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+const Showbooks = () => {
+  const homeStyles = [
+    {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+
+    {
+      display: 'flex',
+      backgroundColor: '#ffffff',
+      margin: '25px',
+      boxShadow: 'rgba(0, 0, 0, 0.37) 0 0 5px 2px',
+    },
+
+    {
+      display: 'flex',
+      flexDirection: 'column',
+      margin: '25px',
+    },
+
+    {
+      display: 'flex',
+      color: 'blue',
+      listStyle: 'none',
+    },
+
+    {
+      width: '4.875rem',
+      height: '1.125rem',
+      opacity: '0.5',
+      fontFamily: 'RobotoSlab',
+      fontSize: '0.813rem',
+      fontWeight: '300',
+      fontStretch: 'normal',
+      fontStyle: 'normal',
+      lineHeight: 'normal',
+      letterSpacing: 'normal',
+      color: 'var(--black-two)',
+    },
+
+    {
+      display: 'flex',
+      flexDirection: 'row',
+      alignItems: 'center',
+      alignSelf: 'baseline',
+      marginLeft: '90px',
+    },
+
+    {
+      color: 'var(--black-two)',
+      width: '7.875rem',
+      height: '1.125rem',
+      margin: '0.063rem 3.625rem 0 3.688rem',
+      opacity: '0.5',
+    },
+  ];
+  return (
+    <section style={homeStyles[1]}>
+      <div style={homeStyles[2]}>
+        <ul style={{ lineHeight: '0' }}>
+          <li style={homeStyles[4]}>
+            Love
+          </li>
+          <li>
+            <h3 style={{ fontSize: '1.375rem' }}>
+              Today will be a good day
+            </h3>
+          </li>
+          <li style={homeStyles[4]}>
+            Jhon Doe
+          </li>
```

Comment:
- [ ] Good work on separating the `Showbook` component from the `book` component but instead of writing book title and author here, you can use it with props. Hint: Takes the book data as a prop and renders the title and the author.